azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/NamespaceType.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.azure.servicebus.management;
+
+/**
+ * Specifies the type of entities the namespace can contain.
+ */
+public enum NamespaceType {
```

Comment:
>NamespaceType [](start = 12, length = 13)

Do we need this enum, especially when it is used only in getters path, not in setters path.

Reply:
Enum would make the client code really easy to use. Since there is an unknown value present, back compat should not be a problem